docs/glossary.md

```diff
@@ -115,11 +131,28 @@ Tickets that reached the end of their window without being called to vote - thes
 
 A mechanism for obtaining free (testnet) coins.
 
+#### Full node
+
+An instance of the Decred daemon `dcrd` which is serving the full version of the blockchain to peers, and fully verifying all of the consensus rules of Decred. 
```

Comment:
See the description of a full node in the dcrd README.  This is not accurate.  Full does not refer to whether or not it serves the full blockchain.

Reply:
Copied the description from the dcrd README